components/admin_console/data_retention_settings/data_retention_settings.tsx

```diff
@@ -226,17 +235,37 @@ export default class DataRetentionSettings extends React.PureComponent<Props, St
             />
         );
     }
-    getCustomPolicyRows = (): Row[] => {
-        return Object.values(this.props.customPolicies).map((policy: any) => {
+    getCustomPolicyRows = (startCount: number, endCount: number): Row[] => {
+        let policies = Object.values(this.props.customPolicies);
+        policies = policies.slice(startCount - 1, endCount);
+
+        return policies.map((policy: DataRetentionCustomPolicy) => {
+            const desciptionId = `customDescription-${policy.id}`;
+            const durationId = `customDuration-${policy.id}`;
+            const appliedToId = `customAppliedTo-${policy.id}`;
+            const menuWrapperId = `customWrapper-${policy.id}`;
             return {
                 cells: {
-                    description: policy.display_name,
-                    channel_messages: this.getMessageRetentionSetting(policy.post_duration !== -1, policy.post_duration),
-                    applied_to: this.getChannelAndTeamCounts(policy),
+                    description: (
+                        <div id={desciptionId}>
+                            {policy.display_name}
+                        </div>
+                    ),
+                    channel_messages: (
+                        <div id={durationId}>
+                            {this.getMessageRetentionSetting(policy.post_duration !== -1, policy.post_duration)}
```

Comment:
We could just pass the post_duration as a param and do the post_duration not equal to -1 within the function